examples/proxy/mountpoint-s3/Dockerfile

```diff
@@ -25,7 +26,7 @@ set -eux
 EOF
 RUN chmod +x /configure_minio.sh
 
-RUN wget https://s3.amazonaws.com/mountpoint-s3-release/1.0.1/x86_64/mount-s3-1.0.1-x86_64.deb -O mount-s3.deb && dpkg -i mount-s3.deb
+RUN wget https://s3.amazonaws.com/mountpoint-s3-release/latest/${TARGETARCH}/mount-s3.deb -O mount-s3.deb && dpkg -i mount-s3.deb
```

Comment:
mountpoint-s3 uses 'x86_64' for amd64 environments.

```diff
+ RUN if [ "$TARGETARCH" = "amd64" ]; then TARGETARCH=x86_64; fi && wget https://s3.amazonaws.com/mountpoint-s3-release/latest/${TARGETARCH}/mount-s3.deb -O mount-s3.deb && dpkg -i mount-s3.deb
```

Reply:
Thanks, too. fixed in eb3fed6